webapp/src/ts/modules/contacts/contacts.component.ts

```diff
@@ -213,31 +213,36 @@ export class ContactsComponent implements OnInit, AfterViewInit, OnDestroy {
     );
   }
 
-  private getUserHomePlaceSummary(homePlaceId?: string) {
+  private getUserHomePlaceSummary() {
     return this.userSettingsService
       .get()
-      .then((userSettings:any) => {
-        const facilityId: string = homePlaceId ?? userSettings.facility_id;
+      .then((userSettings: any) => {
+        const facilityId = Array.isArray(userSettings.facility_id)
+          ? userSettings.facility_id
+          : [userSettings.facility_id];
+
         if (!facilityId) {
           return;
         }
 
         this.globalActions.setUserFacilityId(facilityId);
```

Comment:
Because now this is saving an array. You'll need to update the contact.effects and the contact-component
search by `getUserFacilityId`, you'll find the selectors